src/ui/components/shared/TeamOnboarding.tsx

```diff
@@ -121,6 +121,7 @@ function TeamNamePage({
           onChange={onChange}
           textSize="2xl"
           center={true}
+          theme="dark" // we force onboarding into dark theme
```

Comment:
Seems like you had to hard-code a bunch of colors to dark theme in the CSS, so what affect does this actually have?

Reply:
When you don't pass a theme prop, it defaults to the same css variables we've always use, which change based on dark theme or light theme.

The tricky bit was that we need an ability to force dark theme sometimes, or else the css variables will set light theme and do the wrong thing.

So the hard-coding is the way you get around a css variable making the "light theme text field on a dark theme screen" error seen in the screenshot.